package.json

```diff
@@ -22,23 +22,24 @@
   "bugs": {
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
-  "browserslist": [
```

Comment:
did we want to drop this?

Reply:
Yes, unfortunately this wasn't working with babel. I started an issue thread with one of the maintainers of browserslist here: https://github.com/browserslist/browserslist/issues/484

We'll likely need to duplicate the browserslist if and when we ever need it at this level. Right now we're not using it for anything so I removed it